pkg/templates/template.go

```
@@ -74,3 +75,11 @@
 	t.SetIndexColumn(0)
 	t.Render()
 }
+
+func GetById(templateID string) (*Template, error) {
```

golangci-lint (1.64.8)

[warning] var-naming: func GetById should be GetByID

(revive)